### PR TITLE
test: create test cases

### PR DESCRIPTION
**Pull request by Charm-Xin**
Create some entity classes and repository interfaces under the test package in order to test custom rules

### CODE REVIEW

**Comment by Charm-Xin** [comment]
close https://github.com/ocpddev/jsensible/issues/7

File: build.gradle.kts [hunk 1]
```diff
@@ -40,6 +40,8 @@ dependencies {
     api(libs.archunit)
     testImplementation(kotlin("test"))
     testImplementation(libs.archunit.junit)
+    testImplementation("org.springframework.boot:spring-boot-starter-data-jpa:3.1.0")
```
Comment (EggOxygen) [inline build.gradle.kts]: plz use [spring deps management](https://docs.spring.io/dependency-management-plugin/docs/current/reference/html/) for versioning. Also, now we use gradle version catalog. You should not specify the version number in this file

File: src/test/java/dev/ocpd/jsensible/arch/JpaRulesArchTest.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package dev.ocpd.jsensible.arch
```
Comment (EggOxygen) [inline src/test/java/dev/ocpd/jsensible/arch/JpaRulesArchTest.kt]: Incomplete test cases. It should at least two cases. eg: 'using spring's nullable' and 'using jetbrains's nullable'

**Review by EggOxygen — CHANGES_REQUESTED** [review]

**Comment by EggOxygen** [comment]
You can make this PR draft right now. And change the base to `Charm-Xin:feature/using-spring-nullable` branch.

File: build.gradle.kts (hunk 1 above)
Comment (Charm-Xin) [reply build.gradle.kts]: okay

**Pull request closed without merging**